src/data/fin.lean

```diff
@@ -950,6 +941,12 @@ lemma exists_fin_succ {P : fin (n+1) → Prop} :
 ⟨λ ⟨i, h⟩, fin.cases or.inl (λ i hi, or.inr ⟨i, hi⟩) i h,
   λ h, or.elim h (λ h, ⟨0, h⟩) $ λ⟨i, hi⟩, ⟨i.succ, hi⟩⟩
 
+lemma forall_fin_two {p : fin 2 → Prop} : (∀ i, p i) ↔ p 0 ∧ p 1 :=
+forall_fin_succ.trans $ and_congr_right $ λ _, @unique.forall_iff (fin 1) _ _
+
+lemma exists_fin_two {p : fin 2 → Prop} : (∃ i, p i) ↔ p 0 ∨ p 1 :=
+exists_fin_succ.trans $ or_congr_right $ @unique.exists_iff (fin 1) _ _
```

Comment:
Does it make sense to add `forall_fin_one` and `exists_fin_one` here for convenience? It would be more readable than `@unique.exists_iff (fin 1) _ _`

Reply:
Done.